doc/content/reference/interop-repository/_index.md

```diff
@@ -6,43 +6,43 @@ weight: 3
 
 ## Interoperability Repository
 
-The stack expects the root of interoperability repository to contain `config.yaml`, which contains various interoperablity configuration options:
+The Things Stack expects the root of interoperability repository to contain `config.yml`, which contains various interoperablity configuration options:
 
-```yaml
+```yml
 join-servers:                 # list of Join Server interoperability configurations,
                               # used to map a JoinEUI to the Join Server
-  - file: './path/js.yaml'    # relative path to a file containing Join Server configiration
+  - file: './path/js.yml'    # relative path to a file containing Join Server configiration
     join-euis:                # list of Join EUI prefixes the Join Server should handle
     - '11aa000000000000/16'
 ```
 
-- All paths are relative to the file they're defined in, that is `example/js.yaml` defined in `interopconf/config.yaml` is expected to be located at `interopconf/example/js.yaml`.
-- For `join-servers`, in case Join EUI prefixes overlap, the most specific prefix takes precedence.
-- Interoperability Join Server requests are only performed if either:
-  a. Cluster-local Join Server is not available (e.g. it's disabled in the stack or it's down)
-  b. Cluster-local Join Server is available, but does not contain the information about a device indentified by the given JoinEUI and DevEUI.
-- `0000000000000000/0` prefix matches all JoinEUIs, while being the least specific, hence it always matches last. Use this prefix if you wish to define a "fallback" Join Server interoperability configuration.
+All paths are relative to the file they are defined in, that is `example/js.yml` defined in `interopconf/config.yml` is expected to be located at `interopconf/example/js.yml`.
 
+For `join-servers`, in case Join EUI prefixes overlap, the most specific prefix takes precedence.
 
-For example, consider `config.yaml` as follows:
+Interoperability Join Server requests are only performed if either cluster-local Join Server is not available (e.g. it is disabled in the stack or it is down) or cluster-local Join Server is available, but does not contain the information about a device indentified by the given `JoinEUI` and `DevEUI`.
```

Comment:
I think this is a verbose way of saying that NS tries cluster-local JS first, before contacting external JS via interop

Can you clarify that?

Reply:
https://github.com/TheThingsNetwork/lorawan-stack/pull/1266/commits/043b34b851ab0f9a1f637ff799f50807bcf2bdb1 so this?

Reply:
I would rewrite this paragraph in [jip-and-janneketaal](https://en.wiktionary.org/wiki/jip-en-janneketaal)

Reply:
https://github.com/TheThingsNetwork/lorawan-stack/pull/1266/commits/d8fc38bf57522e3d8ab765f60bf1f63f1ed65f02 ?